sdk/tables/azure-data-tables/inc/azure/data/tables/models.hpp

```diff
@@ -743,6 +743,16 @@ namespace Azure { namespace Data { namespace Tables {
        *
        */
       std::string RowKey;
+      /**
+       * @brief The next Partition key.
+       *
+       */
+      std::string NextPartitionKey;
+      /**
+       * @brief The next row key.
+       *
+       */
+      std::string NextRowKey;
```

Comment:
Should we try to code generation against the rest-api-spec and compare to what we have today, to see if there's other changes we can catch?

https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cosmos-db/data-plane/Microsoft.Tables/preview/2019-02-02/table.json

Reply:
we could i guess, once we're done with the security scramble

Reply:
These match the swagger. Other fields in the swagger are:

* Timeout
* ApiVersion
* ClientRequestId
* DataServiceVersion
* Format
* Top
* Select - Present as SelectColuns
* Filter - Present as Filter
* TableName

Fields that are *not* in the swagger:

* PartitionKey
* RowKey

Note that this is a completely naive scan of the swagger definition. These fields may or may not be relevant.